src/js/index.js

```diff
@@ -1,7 +1,9 @@
 // All helper functions will be imported here, so that they can all be exported within one object.
 import { getBreakpoints, getCurrentScreenWidth } from './modules/breakpointHelper';
+import formReset from './modules/formReset';
 
 export {
+    formReset,
```

Comment:
Would be better if we could make this optional as an export, as it won't be needed on all pages on the site.

Not sure what the best way of doing this would be while enabling easy access to the module from within Fozzie (so people don't have annoying JS absolute paths into module directories).  Any thoughts @DamianMullins @kevinrodrigues @simonsandersje ?

Kinda thinking it would be a little like lodash helpers, where you can import the ones you need as you need them – but equally not sure having a repo for every helper would be the best idea from a maintenance point of view (unless we have a separate folder in fozzie which contains these helpers and we release from these subfolders? Thinking how airbnb do it with the different eslint flavours – they're in one repo, with different package.json's within that repo)

Reply:
I think that's the only way, to have a single repo of helpers in individual folders (containing the index.js, package.json & readme etc) under our fozzie namespace then we can release it all in one go and import them when we need them.